create_dmd_release/patches/dlang.org.patch

```diff
@@ -84,50 +85,32 @@ index b18204f..745fc99 100644
 +dmd-release : $(STD_DDOC) $(DMD_DIR) $(DMD)
-+	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak html $(DDOC_VARS_RELEASE)
++	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak html $(DDOC_VARS_RELEASE_HTML)
  
  dmd-prerelease : $(STD_DDOC_PRE) $(DMD_DIR) $(DMD)
  	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak html $(DDOC_VARS_HTML)
-@@ -425,8 +428,8 @@ druntime-prerelease : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC_PRE)
+@@ -449,8 +453,8 @@ druntime-prerelease : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC_PRE)
  		DOCDIR=${DOC_OUTPUT_DIR}/phobos-prerelease \
  		DOCFMT="$(addprefix `pwd`/, $(STD_DDOC_PRE))"
  
 -druntime-release : ${DRUNTIME_STABLE_DIR} $(DMD_STABLE) $(STD_DDOC)
--	${MAKE} --directory=${DRUNTIME_STABLE_DIR} -f posix.mak target doc $(DDOC_VARS_STABLE) \
+-	${MAKE} --directory=${DRUNTIME_STABLE_DIR} -f posix.mak target doc $(DDOC_VARS_STABLE_HTML) \
 +druntime-release : ${DRUNTIME_DIR} $(DMD) $(STD_DDOC)
-+	${MAKE} --directory=${DRUNTIME_DIR} -f posix.mak target doc $(DDOC_VARS_RELEASE) \
++	${MAKE} --directory=${DRUNTIME_DIR} -f posix.mak target doc $(DDOC_VARS_RELEASE_HTML) \
  	  DOCDIR=${DOC_OUTPUT_DIR}/phobos \
  	  DOCFMT="$(addprefix `pwd`/, $(STD_DDOC))"
  
-@@ -449,9 +452,9 @@ ${DOC_OUTPUT_DIR}/phobos-prerelease/object.verbatim : $(DMD)
+@@ -473,9 +477,9 @@ ${DOC_OUTPUT_DIR}/phobos-prerelease/object.verbatim : $(DMD)
  phobos-prerelease : ${PHOBOS_FILES_GENERATED} $(STD_DDOC_PRE) druntime-prerelease
  	$(MAKE) --directory=$(PHOBOS_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_HTML)
  
 -phobos-release : ${PHOBOS_STABLE_FILES_GENERATED} $(DMD_STABLE) $(STD_DDOC) \
 +phobos-release : ${PHOBOS_FILES_GENERATED} $(DMD) $(STD_DDOC) \
  		druntime-release dmd-release
--	$(MAKE) --directory=$(PHOBOS_STABLE_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_STABLE)
-+	$(MAKE) --directory=$(PHOBOS_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_RELEASE)
+-	$(MAKE) --directory=$(PHOBOS_STABLE_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_STABLE_HTML)
++	$(MAKE) --directory=$(PHOBOS_DIR_GENERATED) -f posix.mak html $(DDOC_VARS_RELEASE_HTML)
  
  phobos-prerelease-verbatim : ${PHOBOS_FILES_GENERATED} ${DOC_OUTPUT_DIR}/phobos-prerelease/index.verbatim
  ${DOC_OUTPUT_DIR}/phobos-prerelease/index.verbatim : verbatim.ddoc \
-diff --git a/std_navbar-release.ddoc b/std_navbar-release.ddoc
-index 3712f92..34a3dad 100644
---- a/std_navbar-release.ddoc
-+++ b/std_navbar-release.ddoc
-@@ -2,12 +2,7 @@ SUBNAV=
- $(SUBNAV_TEMPLATE
-     $(DIVC head,
-         $(H2 Library Reference)
--        $(P $(SPANC smallprint, version $(LATEST) $(SPANC separator, $(BR))
--            switch to
--            $(LINK2 ../phobos-prerelease/index.html, pre-release). $(BR)
--            $(LINK2 ../library/index.html, preview) beta library reference.
--            )
--        )
-+        $(P $(SPANC smallprint, version $(LATEST)))
-         $(P $(LINK2 index.html, overview))
```

Comment:
No longer needed with the new version-changer widget (which is JS and shouldn't run in chm).